docs/source/physical_robot_setup/index.rst

```diff
@@ -0,0 +1,135 @@
+
+----------------------------------
+Preparation + Install the Hardware
+----------------------------------
+Instructions about the assembly of robots can be found in the *how-to-build-a-modular-robot-guide*.
+
+Firstly make sure to have all the materials necessary to build the physical robot.
+
+For any type of hardware, there are some required items:
+
+* A housing that will contain the rest of the core hardware. Models to print the core housing can be found on the following git repo: `<https://github.com/ci-group/revolve-models>`_.
+* A Raspberry Pi (RPi). For V2, this must be a RPi 4.
+* A HAT (hardware attached on top), as described in the following hardware specific sections.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Hardware setup for a V1 Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+**V1 is only available to CI Group.**
+
+
+For the V1 Robot some specific Hardware is required:
+
+* It is required to use a V1 HAT, available in the CI Group lab.
+* A V1 specific battery, available in the CI Group lab.
+
+Attach the HAT onto the RPi and put them into the housing.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Hardware setup for a V2 Robot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For instructions on  how to set up the hardware of a V2 core, refer to `<https://github.com/ci-group/robohat>`_.
+
+------------------
+Setting up the RPi
+------------------
+This step is the same for all types of hardware.
+To set up the RPi follow the steps below carefully.
+
+#. Flash the SD card with the RPi OS (previously Raspbian). Some Important notes:
+
+    * If you are required to pick a username and password, it is recommended to pick :code:`pi` and :code:`raspberry` for easier access. These are the defaults for most RPi installations.
+    * Reminder for CI Group people: if you select a keyboard layout, pick `English(US)`. It can always be changed using the `raspi-config` tool on the RPi.
+    * When setting up the Wi-Fi connection, select your Country. (CI-Group Wi-Fi: *ThymioNet*)
+
+#. Boot the RPi and get command line access. Here are some general Tips:
+
+    * You can connect a keyboard and screen.
+    * Better is to SSH. This works only if you and the RPi are on the same network. For CI Group, connect it to *ThymioNet* Wi-Fi.
+    * If you want to SSH and don't know the IP of the RPi, you can use: :code:`sudo nmap -sP <your ip>` to find all clients on your network.
+
+---------------------------
+Install Revolve2 on the RPi
+---------------------------
+Setting up Revolve2 on the robot requires different steps, depending on the hardware version. Some general steps for all versions:
+
+#. Check if pyenv is installed by default on your RPi. If not follow the next points:
+#. Set up a global pyenv. This is to prevent changes to the system's Python installation.:
+
+    #. Install pyenv: :code:`curl https://pyenv.run | bash` & :code:`git clone https://github.com/pyenv/pyenv-virtualenv.git $(pyenv root)/plugins/pyenv-virtualenv`
```

Comment:
Pyenv+virtualenv is an extension of pyenv. Earlier i meant to say that the extension might come by default with pyenv. I will check that today. If so, we can remove the second part of this step